lib/pbench/agent/results.py

```diff
@@ -278,6 +278,13 @@ def make_result_tb(self, single_threaded: bool = False) -> TarballRecord:
         return TarballRecord(name=tarball, length=tar_len, md5=tar_md5)
 
 
+class ResponseStatus:
+    def __init__(self, status_code: int, reason: str, message: str):
+        self.status_code = status_code
+        self.reason = reason
+        self.message = message
```

Comment:
Might as well remove this if you're not using it anymore.

Reply:
Yup.